cmd/broker/provisioning.go

```diff
@@ -84,6 +84,13 @@ func NewProvisioningProcessingQueue(ctx context.Context, provisionManager *proce
 			stage:     createRuntimeStageName,
 			step:      provisioning.NewGenerateRuntimeIDStep(db.Operations(), db.RuntimeStates(), db.Instances()),
 		},
+		// postcondition: operation.RuntimeID is set
+		{
+			condition: provisioning.SkipForOwnClusterPlan,
```

Comment:
we do not need this condition, it should wokr also for own cluster

Reply:
In the next commit.

Reply:
But then test for OwnCluster fail.
Need to investigate.

Reply:
Removed no longer needed step.